.vscode/bevy.code-snippets

```diff
@@ -0,0 +1,75 @@
+{
+    "Bevy: New module-level function Plugin": {
+        "scope": "rust",
+        "prefix": "plugin_fn",
+        "body": [
+            "use bevy::prelude::*;",
```

Comment:
Remove this line and use `"requires": "bevy::prelude::*"` instead. Add that to all other snippets as well.

Reply:
Woah

Reply:
I get a "Property requires is not allowed" warning, and I don't see this mentioned in the docs.

Reply:
Got it from the [rust-analyzer docs](https://rust-analyzer.github.io/manual.html#user-snippet-completions)

Reply:
Update from Discord: the rust-analyzer snippet syntax does *not* work.